data/korath/korath missions.txt

```diff
@@ -257,3 +257,223 @@ mission "Small Scale Delivery"
+				goto title
+
+			label refuse
+			`	You see the leader of the group visibly sag as the Quarg translates your words. Two younger Korath bear him up as the group turns and leaves.`
+				decline
+
+			label pay
+			`	A ripple of what appears to be dissatisfaction passes through the group, but their leader silences them with a gesture.`
+
+			label title
+			`	The leader turns and speaks to you directly. His eyes never waver from yours even as you wait for the translation. "Many ship skills work parent-child to learn. For some families, three generations are lost. We will not try to salvage the body box of our kin, for none can bear to fly it. We offer you title to salvage the <EDC-ship> and carry our grief far from here."`
+				accept
+
+	on stopover
+		conversation
+			`You enter the <EDC-ship> wearing environment suits. It's been several days since the crash and Korath ships run hot. This will be an upleasant task in the suits and perhaps impossible without them.`
+			`	While your engineer works on refreshing the atmosphere with outsite air, the rest of the crew gathers the bodies and places them in the fibrous, vaguely egg-shapped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
+			`	You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.`
+			`	The crew makes a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
+
+	on complete
+		conversation
+			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch. You count the coffins together and show them the boxes of possessions.`
+			`	The officer thanks you and hands you a data chip that vaguely reminds you of a license.`
+			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cachophony of barking sounds. Some Korath try to rush forward but are held back by others, you presume their family.`
+			`	You go back inside your ship rather than intrude on their grief. You slot the data chip into your systems but the information is in Korathi. You set it aside and work on a salvage plan.`
+			`	As you review your notes on the ship and the number of crew they were carrying, it's clear that you will need to carry at least 66 additional crew that you can communicate with to be able to salvage the <EDC-ship>. The closest system where you can find that many Human speakers is on Greeenwater in Hai space.`
+
+mission "Efreti: Distress Call 3"
```

Comment:
Since we're picking up the 66 people in this mission and you mentioned the requirement in the previous mission, it probably makes sense to have the bunks requirement begin in this mission.